tests/10-migration-backend-tests/verification/verify_migrations_test.go

```diff
@@ -203,6 +266,42 @@ func Test_VerifyTaxonomyCopyrightAndUse(t *testing.T) {
 	}
 }
 
+func Test_VerifyTaxonomyTermResourceType(t *testing.T) {
+	expectedJson := ExpectedResourceType{}
+	unmarshalJson(t, "taxonomy-resourcetypes.json", &expectedJson)
+
+	// sanity check the expected json
+	assert.Equal(t, "taxonomy_term", expectedJson.Type)
+	assert.Equal(t, "resource_types", expectedJson.Bundle)
+
+	u := &JsonApiUrl{
+		t:            t,
+		baseUrl:      DrupalBaseurl,
+		drupalEntity: expectedJson.Type,
+		drupalBundle: expectedJson.Bundle,
+		filter:       "name",
+		value:        expectedJson.Name,
+	}
+
+	// retrieve json of the migrated entity from the jsonapi and unmarshal the single response
+	res := &JsonApiResourceType{}
+	u.get(res)
```

Comment:
If only one result is expected, I'd prefer the use of `getSingle` over `get`.

Reply:
I'll change this to `getSingle`.